src/uu/tail/src/follow/watch.rs

```diff
@@ -91,17 +90,21 @@ pub struct WatcherService {
     /// platform specific event driven method. Since `use_polling` is subject to
     /// change during runtime it is moved out of [`Settings`].
     pub use_polling: bool,
+
     pub watcher_rx: Option<WatcherRx>,
     pub orphans: Vec<PathBuf>,
     pub files: FileHandling,
+
+    pub pid: platform::Pid,
 }
 
-impl WatcherService {
+impl Observer {
```

Comment:
Any particular reason for this name change?

Reply:
Oops guess I start sounding like a broken record. I agree on this one, but take care not to introduce too many unnecessary changes.

Reply:
No problem :) I know what you mean, but this struct was introduced in a previous pr and I thought you agree on this change. I usually don't rename at all but this was a struct I've introduced myself...